pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -166,7 +166,7 @@ func NewNodePerformance(assetsDir string, profile *performancev2.PerformanceProf
 		}
 	}
 
-	profileData, err := getProfileData(getProfilePath(components.ProfileNamePerformance, assetsDir), templateArgs)
+	profileData, err := getProfileData(filepath.Join("tuned", components.ProfileNamePerformance), templateArgs)
```

Comment:
maybe use a constant instead of "tuned" string

Reply:
I think it is a single place, so not really worth it, but if you insist I can do it.